### DISCUSSION TITLE
Does this support claude models?

### DISCUSSION THREAD

**Discussion by CodingPA** [ChatGPTNextWeb/NextChat#4584] [Q&A]
I see in the env variables references to an anthropic key and models -- are we able to chat with claude 3 opus?  i added in my api keys and the other env variables, but get 'The model does not exist or you do nto have access to it' messages -- I can make calls to these models in other scripts with this API key?  Is there any documentation on how to use claude 3 in anthropic with this chat?  i am deploying with docker.  All I see in the readme is a reference to a key and nothing else.  Thank you!

**Comment by llccing** [comment]
Hi, I have same question. is there any token to configure to set claude model. For me openai works, but claude not works. i can directly call the proxy api by postman to use claude, but i can use it in this tool.
